dynamiq/nodes/tools/jina.py

```diff
@@ -0,0 +1,242 @@
+import enum
+from typing import Any, ClassVar, Literal
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from dynamiq.connections import Jina
+from dynamiq.nodes import NodeGroup
+from dynamiq.nodes.node import ConnectionNode, ensure_config
+from dynamiq.runnables import RunnableConfig
+from dynamiq.utils.logger import logger
+
+
+class JinaScrapeInputSchema(BaseModel):
+    url: str = Field(..., description="Parameter to provide a url of the page to scrape.")
+
+
+class JinaResponseFormat(str, enum.Enum):
+    DEFAULT = "default"
+    MARKDOWN = "markdown"
+    HTML = "html"
+    TEXT = "text"
+    SCREENSHOT = "screenshot"
+    PAGESHOT = "pageshot"
+
+
+class JinaScrapeTool(ConnectionNode):
+    """
+    A tool for scraping web pages, powered by Jina.
+
+    This class is responsible for scraping the content of a web page using Jina.
+
+    Attributes:
+        group (Literal[NodeGroup.TOOLS]): The group to which this tool belongs.
+        name (str): The name of the tool.
+        description (str): A brief description of the tool.
+        SCRAPE_PATH(str): The constant path to perform scrape request using Jina API.
+        connection (Jina): The connection instance for the Jina API.
+        timeout(int): The timeout of the scraping process.
+        input_schema (JinaSearchInputSchema): The input schema for the tool.
+    """
+    group: Literal[NodeGroup.TOOLS] = NodeGroup.TOOLS
+    name: str = "Jina Scraper Tool"
+    description: str = (
+        "A tool for scraping web pages, powered by Jina. " "You can use this tool to scrape the content of a web page."
+    )
+    SCRAPE_PATH: ClassVar[str] = "https://r.jina.ai/"
```

Comment:
Please move this ClassVar and divide with blank line from other attrbitutes. Let's make this looks good for read